applications/DEMApplication/custom_utilities/multiaxial_control_module_generalized_2d_utilities.hpp

```diff
@@ -0,0 +1,367 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ignasi de Pouplana
+//                   Guillermo Casas
+//
+
+
+#ifndef KRATOS_MULTIAXIAL_CONTROL_MODULE_GENERALIZED_2D_UTILITIES
+#define KRATOS_MULTIAXIAL_CONTROL_MODULE_GENERALIZED_2D_UTILITIES
+
+// /* External includes */
+
+// System includes
+
+// Project includes
+#include "includes/variables.h"
+
+/* System includes */
+#include <limits>
+#include <iostream>
+#include <iomanip>
+#include <map>
+
+/* External includes */
+#ifdef _OPENMP
+#include <omp.h>
```

Comment:
I recommend using the [parallel_utilities](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/parallel_utilities.h) instead

Reply:
We will switch to the parallel_utilities in the future, but I think this is not urgent.